packages/scaffolding-cli/src/domain/config/static.config.json

```diff
@@ -287,6 +270,49 @@
       }
     ]
   },
+  "ssrGkeJenkins": {
+    "gitRepo": "",
+    "gitRef": "",
+    "localPath": "",
+    "folderMap": [
+      {
+        "src": "shared/aux_scripts",
+        "dest": "./aux_scripts"
+      },
+      {
+        "src": "shared/README.md",
+        "dest": "./README.md"
+      },
+      {
+        "src": "shared/_gitignore",
+        "dest": "./.gitignore"
+      },
+      {
+        "src": "build/jenkins/gcp/ssr-gke.Jenkinsfile",
+        "dest": "build/jenkins/gcp/Jenkinsfile"
+      },
+      {
+        "src": "deploy/azure/app/kube",
+        "dest": "deploy/gcp/app"
+      },
+      {
+        "src": "deploy/k8s/app/base_gke-app-deploy.yml",
+        "dest": "deploy/k8s/app/base_gke-app-deploy.yml"
+      },
+      {
+        "src": "test",
+        "dest": "test"
```

Comment:
Do you think we should remove the testcafe and selenium frameworks?

Reply:
why?